### PR TITLE
[ci] migrate to new Dockerfile

### PR DESCRIPTION
**Pull request by StrikerRUS**

### CODE REVIEW

File: .vsts-ci.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ variables:
 resources:
   containers:
   - container: ubuntu1404
-    image: lightgbm/vsts-agent:ubuntu-14.04
+    image: lightgbm/vsts-agent:latest
```
Comment (guolinke) [inline .vsts-ci.yml]: sorry,  I think the tag name should be changed to ubuntu-14.04.
I will fix it in docker hub

Comment (StrikerRUS) [reply .vsts-ci.yml]: Got it!

Comment (StrikerRUS) [reply .vsts-ci.yml]: @guolinke Done! Can we remove unused tags on Docker Hub?

Comment (StrikerRUS) [reply .vsts-ci.yml]: I mean `latest` and `test`.

Comment (guolinke) [reply .vsts-ci.yml]: sure

**Pull request merged**